grafana/dashboards/brigade.json

```diff
@@ -4,10 +4,25 @@
   "uid": "xDvsAAR7k",
   "title": "Brigade",
   "version": 4,
-  "refresh": "5s",
+  "refresh": "2s",
   "style": "dark",
   "editable": false,
   "graphTooltip": 0,
+  "timepicker": {
+    "refresh_intervals": [
+      "2s",
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ]
+  },
```

Comment:
I don't think we really need to change this. If Grafana refreshes at most every 5 second, that's fine. What I really wanted was more points on the curves to improve how closely it reflects reality.

Reply:
Oh I see, I think I might have misunderstood your request. Let me take a look at how to plot more points on a Grafana time series.